tidb-cloud/setup-self-hosted-kafka-pls.md

```diff
@@ -0,0 +1,528 @@
+---
+title: Setup Self-hosted Kafka Private Link Service in AWS
+summary: This document explains how to set up private link service for self-hosted Kafka in AWS and how to make it work with TiDB Cloud.
+---
+
+# Setup Self-hosted Kafka Private Link Service in AWS
```

Comment:
I think this document is too long. If the kafka service is already deployed, it's hard to extract efficient information to know how to integrate with tidb cloud by private link for existing kafka service. So I think we can divide to two documents, and the one is setup private link with new-deployed kafka service, and the other is setup private link with existing kafka service.

Reply:
Yes, you are right.

Reply:
restructure the doc as following:
### Prerequisites
### First, Setup Kafka Cluster
#### Deploy a New Kafka Cluster
#### Reconfigure a Running Kafka Cluster
### Second, Expose Kafka Cluster as Private Link Service
### Third, Connect from TiDB Cloud